ProductionModePlugin.js

```diff
@@ -129,8 +129,16 @@ ProductionModePlugin.prototype.apply = function(compiler) {
         return /globalize-compiled-data/.test(chunk.name);
       });
       allModules.forEach(function(module) {
+        var chunkRemoved, chunk;
         if (globalizeCompilerHelper.isCompiledDataModule(module.request)) {
           hasAnyModuleBeenIncluded = true;
+          while (module.chunks.length) {
+            chunk = module.chunks[0];
+            chunkRemoved = module.removeChunk(chunk);
+            if (!chunkRemoved) {
+              throw new Error("Failed to remove chunk " + chunk.id + " for module " + module.request);
+            }
+          }
           module.chunks.forEach(function(chunk) {
             module.removeChunk(chunk);
           });
```

Comment:
I thought you would replace the `forEach` block with the `while` one. Forgot to remove this? :)